package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1064,6 +1068,57 @@ def write_next_timestep(self, ts=None, **kwargs):
         self._check_pdb_coordinates()
         self._write_timestep(ts, **kwargs)
 
+    @staticmethod
+    def _deduce_PDB_atom_name(atom):
+        """Deduce how the atom name should be aligned.
+
+        Atom name format can be deduced from the atom type, yet atom type is
+        not always available. This function uses the atom name and residue name
+        to deduce how the atom name should be formatted. The rules in use got
+        inferred from an analysis of the PDB. See gist at
+        <https://gist.github.com/jbarnoud/37a524330f29b5b7b096> for more
+        details.
+        """
+        ions = ('FE', 'AS', 'ZN', 'MG', 'MN', 'CO', 'BR',
+                'CU', 'TA', 'MO', 'AL', 'BE', 'SE', 'PT',
+                'EU', 'NI', 'IR', 'RH', 'AU', 'GD', 'RU')
```

Comment:
I already noticed this in your notebook. Why is `CA` missing as an ion? Wasn't that the example you used in your issue?